internal/web/sse/ssehub.go

```diff
@@ -0,0 +1,93 @@
+package sse
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"strings"
+	"sync"
+)
+
+// sseHub maintains the set of active clients and broadcasts messages to the clients.
+type sseHub struct {
+	// Inbound messages from the clients.
+	broadcast chan *sse
+
+	// register requests from the clients.
+	register chan *sseClient
+
+	// unregister requests from clients.
+	unregister chan *sseClient
+
+	// Registered clients.
+	clients sync.Map // map[*sseClient]struct{}
+}
+
+type sse struct {
+	event string
+	data  string
```

Comment:
I noticed that this is never used. can it be removed?

Reply:
It's used in the `sseHub` struct for the broadcasting channel, and `sse` objects are also created in the `broadcaster`

Reply:
Sorry, I was referring only to the `data` property.

Reply:
Oh, good point. Originally we sent the rendered html in the `data` property, but currently we don't use it because we only use the `event` to define the trigger. 
I'm unsure, but I would still leave it in, because otherwise that would be an SSE Hub that couldn't send actual data.

Reply:
OK, let's keep it then. I just wanted to bring it up